tools/build-tools/src/bumpVersion/README.md

```diff
@@ -83,7 +83,7 @@ Creating release 0.16.1
     Waiting for package to publish @fluid-example/canvas@0.16.1...
 ...cut for brevity...
     Waiting for package to publish @fluidframework/tool-utils@0.16.1...
-    Waiting for package to publish @fluidframework/odsp-utils@0.16.1...
+    Waiting for package to publish @fluidframework/odsp-doclib-utils@0.16.1...
```

Comment:
this one probably shouldn't be changed

Reply:
because it is pointing to version when it was of other name?

Reply:
hmm, makes sense